src/json-data.js

```diff
@@ -0,0 +1,57 @@
+const fs = require('fs');
+const path = require('path');
+const root = path.resolve(process.cwd(), 'src', 'stache', 'data');
+const reserved = require('reserved-words');
+
+const isPropertyNameValid = (propertyName) => {
+  if (!propertyName) {
+    return false;
+  }
+
+  if (propertyName === 'prototype') {
+    return false;
+  }
+
+  // Parsing as boolean because reserved-words returns `undefined` for falsy values.
+  return !reserved.check(propertyName, 'es6', true);
```

Comment:
This is a bit confusing. Is it returning true if it's found and then converted to false, but then if it's not found it's getting returned as undefined and then converting that falsy value to true? I suppose there isn't a way around it because of the API.